static/scripts/app-register.js

```diff
@@ -1,7 +1,55 @@
 const formbtn = document.querySelector('.btn');
+const formInputs = document.querySelectorAll('.form-inputs');
 // http://localhost:2228/register
 // const inputs = document.querySelectorAll('.form-inputs');
 
+function keyDownEvents(){
+    const REGEXP = regExpDelivery();
+
+    for(let i = 0; i < formInputs.length; i++){
+
+        formInputs[i].addEventListener('input', () => {
+            
+            let user_err = document.createElement('small');
+            if(!REGEXP[i].test(formInputs[i].value)){
+                // console.log(formInputs[i].nextElementSibling.nodeName)
```

Comment:
@FirstDayAtWork We should remove this at some point, not a blocker for now.

Reply:
> @FirstDayAtWork We should remove this at some point, not a blocker for now.